webapp/helpers.py

```diff
@@ -25,17 +25,30 @@ def __init__(self, url_map, *items):
 
 def get_licenses():
     try:
-        with open("webapp/licenses.json") as f:
```

Comment:
@Lukewh So can we delete this file now?

Reply:
Ah yes - I'll comment it out with an extra comment - I don't want to remove it because I'm clinging on to the possibility that we'll soon move to v3 and this will be relevant once again

Reply:
oh I see - I DID delete it. Yes we can delete this for now. We'll need to put it back when we can use v3